### ISSUE TITLE
[BUG] Crash when clicking on scrollbar

### ISSUE THREAD

**Issue by DareFox** [DareFox/SaveDTF-Compose#15]
### Program version

pre1--2.0.0

### Error description

```markdown
Exception in thread "AWT-EventQueue-0" java.lang.IllegalArgumentException: Index should be non-negative (-1)
```

### How to reproduce error

```markdown
Click on scrollbar
```

### Expected behavior

```text
Scroll to specified position
```

### Screenshots

![image](https://user-images.githubusercontent.com/47672780/178123205-82aba08c-c648-453f-a831-0d8e48e45411.png)

### Information about the device

OS: Linux Mint 20.3 x86_64 
Kernel: 5.4.0-121-generic 

### Additional context

```markdown
00:26:39.328 [AWT-EventQueue-0] INFO  ui.composables.CheckVersion - Checking updates
00:26:40.103 [AWT-EventQueue-0] INFO  ui.composables.CheckVersion - GitHub version: 1.0.4
00:26:40.104 [AWT-EventQueue-0] INFO  ui.composables.CheckVersion - Current version: 2.0.0
Exception in thread "AWT-EventQueue-0" java.lang.IllegalArgumentException: Index should be non-negative (-1)
	at androidx.compose.foundation.lazy.list.LazyListScrollPosition.update-NTjDD6g(LazyListScrollPosition.kt:100)
	at androidx.compose.foundation.lazy.list.LazyListScrollPosition.requestPosition-NTjDD6g(LazyListScrollPosition.kt:83)
	at androidx.compose.foundation.lazy.LazyListState.snapToItemIndexInternal$foundation(LazyListState.kt:199)
	at androidx.compose.foundation.lazy.LazyListState$scrollToItem$2.invokeSuspend(LazyListState.kt:194)
	at androidx.compose.foundation.lazy.LazyListState$scrollToItem$2.invoke(LazyListState.kt)
	at androidx.compose.foundation.lazy.LazyListState$scrollToItem$2.invoke(LazyListState.kt)
	at androidx.compose.foundation.gestures.DefaultScrollableState$scroll$2$1.invokeSuspend(ScrollableState.kt:148)
	at androidx.compose.foundation.gestures.DefaultScrollableState$scroll$2$1.invoke(ScrollableState.kt)
	at androidx.compose.foundation.gestures.DefaultScrollableState$scroll$2$1.invoke(ScrollableState.kt)
	at androidx.compose.foundation.MutatorMutex$mutateWith$2.invokeSuspend(MutatorMutex.kt:160)
	at androidx.compose.foundation.MutatorMutex$mutateWith$2.invoke(MutatorMutex.kt)
	at androidx.compose.foundation.MutatorMutex$mutateWith$2.invoke(MutatorMutex.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:89)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:264)
	at androidx.compose.foundation.MutatorMutex.mutateWith(MutatorMutex.kt:153)
	at androidx.compose.foundation.gestures.DefaultScrollableState$scroll$2.invokeSuspend(ScrollableState.kt:145)
	at androidx.compose.foundation.gestures.DefaultScrollableState$scroll$2.invoke(ScrollableState.kt)
	at androidx.compose.foundation.gestures.DefaultScrollableState$scroll$2.invoke(ScrollableState.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:89)
	at kotlinx.coroutines.CoroutineScopeKt.coroutineScope(CoroutineScope.kt:264)
	at androidx.compose.foundation.gestures.DefaultScrollableState.scroll(ScrollableState.kt:144)
	at androidx.compose.foundation.gestures.ScrollableState$DefaultImpls.scroll$default(ScrollableState.kt:51)
	at androidx.compose.foundation.lazy.LazyListState.scrollToItem(LazyListState.kt:193)
	at androidx.compose.foundation.LazyScrollbarAdapter.snapTo(Scrollbar.desktop.kt:479)
	at androidx.compose.foundation.LazyScrollbarAdapter.scrollTo(Scrollbar.desktop.kt:453)
	at androidx.compose.foundation.Scrollbar_desktopKt$scrollOnPressOutsideSlider$1$1.invokeSuspend(Scrollbar.desktop.kt:321)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at androidx.compose.ui.platform.FlushCoroutineDispatcher$dispatch$2$1.invoke(CoroutineDispatchers.skiko.kt:51)
	at androidx.compose.ui.platform.FlushCoroutineDispatcher$dispatch$2$1.invoke(CoroutineDispatchers.skiko.kt:46)
	at androidx.compose.ui.platform.FlushCoroutineDispatcher.performRun(CoroutineDispatchers.skiko.kt:78)
	at androidx.compose.ui.platform.FlushCoroutineDispatcher.access$performRun(CoroutineDispatchers.skiko.kt:29)
	at androidx.compose.ui.platform.FlushCoroutineDispatcher$dispatch$2.invokeSuspend(CoroutineDispatchers.skiko.kt:46)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:716)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:741)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)

Process finished with exit code 0
```

**Comment by DareFox** [comment]
Bug appeared somewhere between efdef73 (non-inclusive) and 8871539 commits

Diff: [efdef73..8871539](https://github.com/DareFox/SaveDTF-Compose/compare/efdef73..8871539)

**Comment by DareFox** [comment]
Found source of the bug: 292873c

It seems that covering a scrollbar with `Box` creates this bug

**Comment by DareFox** [comment]
Closed by d1b4440
